Userland/Libraries/LibJIT/Assembler.h

```diff
@@ -115,18 +115,30 @@ struct Assembler {
         SignedGreaterThanOrEqualTo = 0xD,
         SignedLessThan = 0xC,
         SignedLessThanOrEqualTo = 0xE,
+        SignedOverflow = 0x0,
+        NotSignedOverflow = 0x1,
     };
 
     static constexpr u8 encode_reg(Reg reg)
     {
         return to_underlying(reg) & 0x7;
     }
 
-    enum class Patchable {
+    enum class Patchable : u8 {
         Yes,
         No,
     };
 
+    enum class ArithSign : u8 {
+        Positive,
+        Negative
+    };
+
+    enum class OpBits : u8 {
+        Bits64 = 1,
+        Bits32 = 0
+    };
```

Comment:
This is not a true replacement for a rex_w enum, as some instructions are inherintly 64 bit, so we'd now pass `Bits32`